templates/common/infra/bicep/core/host/functions-dotnet.bicep

```diff
@@ -0,0 +1,36 @@
+param environmentName string
```

Comment:
Also update the README for Azure SQL instead of Cosmos

Reply:
Resolved in latest commit.